src/libraries/System.Text.Json/tests/System.Text.Json.FSharp.Tests/CollectionTests.fs

```diff
@@ -175,3 +175,11 @@ let ``Map async deserialization should be supported``() = async {
 
     Assert.Equal<Map<string, int>>(inputs, result)
 }
+
+[<Fact>]
+let ``Map deserialization should reject duplicate keys``() =
+    Assert.Throws<JsonException>(fun () -> JsonSerializer.Deserialize<Map<string, int>>("""{ "1": "a", "1": "a" }""") |> ignore) |> ignore
```

Comment:
I'm surprised that this would throw in the default case. Shouldn't it only happen when `AllowDuplicateProperties` has been disabled?

Reply:
The target type `int` is wrong, updated test